elasticsearch/utils/es_seed_acl

```diff
@@ -23,6 +23,7 @@ function sgadmin {
     -tst JKS \
     -tspass tspass \
     -nhnv \
+    -arc \
```

Comment:
Should this be conditional, or is it ok to ignore red cluster at all times?

Reply:
I was wondering the same thing but I can't honestly say.  Red simply means you dont have placed replicas for the cluster as a whole.  ES should be smart enough to figure out how to reconcile changes if there are multiple replicas for this particular index.

Reply:
The SG admin tool is also smart enough that it won't try to create this index if it already exists.